cirq-core/cirq/ops/linear_combinations_test.py

```diff
@@ -1156,6 +1156,23 @@ def test_pauli_sum_pow():
     for psum in [psum1, psum2, psum3, psum4]:
         assert cirq.approx_eq(psum**0, identity)
 
+    # tests for exponents greater than two for both even and odd
+    psum5 = cirq.Z(q0) * cirq.Z(q1) + cirq.Z(q2) + cirq.Z(q3)
+    correctresult = psum5.copy()
+    for e in range(1,9):
+        powfunctionresult = psum5**e
+        assert correctresult == powfunctionresult
```

Comment:
Nit: I suggest we do here without the local variable:
```
assert correctresult == psum5**e
```

Reply:
Sure, no need for having a local variable